packages/push/src/instruction/float.rs

```diff
@@ -13,8 +19,15 @@ use crate::{
 #[derive(Debug, strum_macros::Display, Copy, Clone, EnumIter, Eq, PartialEq)]
 #[non_exhaustive]
 pub enum FloatInstruction {
-    #[strum(to_string = "Push({0})")]
-    Push(OrderedFloat<f64>),
+    Pop(Pop<OrderedFloat<f64>>),
+    #[strum(to_string = "{0}")]
+    Push(PushValue<OrderedFloat<f64>>),
+    Dup(Dup<OrderedFloat<f64>>),
+    Swap(Swap<OrderedFloat<f64>>),
+    IsEmpty(IsEmpty<OrderedFloat<f64>>),
+    StackDepth(StackDepth<OrderedFloat<f64>>),
+    Flush(Flush<OrderedFloat<f64>>),
```

Comment:
As I've mentioned on stream before we should probably implement `From<>` for all of these.

Reply:
Agreed. That would happen separately in each instruction type (e.g., `IntInstruction`, `FloatInstruction`, etc.), correct?

Would this be a reason to have a `Common` enum? Then we could have a single `impl From<Common<i64>> for IntInstruction` instead of having a dozen separate `From` implementations in each type. Do we think that would work?

Reply:
I would, for now, even if it is a lot of copy pasta just add individual from implementations for each. Yes a common enum might help reduce the clutter but I'd say in the long term the from implementations should probably just be auto generated by some kind of common infrastructure, and a common enum brought up other issues with EnumIter that I'd just like to sidestep at least for now